File: Assignments September 2024/Prog72t.rb
puts "First time in military time"
a = gets.chomp.to_f
puts "Second time in military time"
b = gets.chomp.to_f
a = a / 100
b = b / 100
c = a - b
if c > 0
    c = c
elsif c < 0
    c = c * -1
end

puts "Time difference is: " + c.to_s + " Hours"

File: Assignments September 2024/Prog72t.py
#Military time.
a = int(input("insert start time: "))
b = int(input("Insert end time: "))
c = a / 100
d = b / 100

if a > b:
    diff = c - d #1730 - 900 = 830 need to translate to 8:30
    print("The time difference is: " + str(diff) + " Hours")
elif a < b:
    diff = d - c
    print("The time difference is: " + str(diff) + " Hours")